### PR TITLE
Updated VEP and VR examples for chicken(GRCg7b) for e107

### PR DESCRIPTION
**Pull request by nakib103**

### CODE REVIEW

**Comment by ens-ma7** [comment]
Hi @nakib103 any update on this PR? Is it ready for review? If it is can you please get someone to review and approve it and just comment in there once you are ready for us to merge it.

**Review by dglemos — APPROVED** [review]

**Pull request merged**
